core/trino-spi/src/main/java/io/trino/spi/StandardErrorCode.java

```diff
@@ -189,6 +189,7 @@
     EXCEEDED_TASK_DESCRIPTOR_STORAGE_CAPACITY(131082, INSUFFICIENT_RESOURCES),
 
     UNSUPPORTED_TABLE_TYPE(133001, EXTERNAL),
+    METADATA_NOT_FOUND(133002, EXTERNAL),
```

Comment:
this is part of `Introduce dropTable SPI method that takes SchemaTableName` commit

i don't think it belongs here. Also, i am not convinced we need a new error code at all

Reply:
Initially, I thought of using common error code which will be thrown from `IcebergMetadata` and `DeltaLakeMetadata`. Catch that error code in `DropTableTask` and force drop if error code matches with `METADATA_NOT_FOUND`.

Now the approach has been changed. I have removed this error code.